src/components/calcite-slider/calcite-slider.tsx

```diff
@@ -758,14 +758,21 @@ export class CalciteSlider {
    * expensive operations consider using a debounce or throttle to avoid
    * locking up the main thread.
    */
+  @Event() calciteSliderInput: EventEmitter;
+
+  /**
+   * Fires on when the thumb is released on slider
```

Comment:
The change event should also fire when the value is updated via keyboard. The only difference between this and the input event is that change event does not fire between the thumb being pressed and released: https://codepen.io/jcfranco/pen/VwWMjBP